packages/@o3r/store-sync/src/sync-storage/interfaces.ts

```diff
@@ -16,6 +16,8 @@ export interface SyncStorageSyncOptions {
   filter?: string[];
   /** Spacing for serializer */
   space?: string | number;
+  /** Defines whether the state is be synced at feature level*/
```

Comment:
I think the comment is a bit unclear regarding the purpose of this configuration.
I would suggest something like `/** Defines whether the store is loaded as forFeature */`

Reply:
Changed the comment as suggested.